apps/demo/src/app/cart/components/cart-items/cart-items.component.scss

```diff
@@ -0,0 +1,11 @@
+@import 'utilities';
+@import 'theme';
+
+.demo-cart-items {
+	display: block;
+
+	&__content {
+		padding: 25px 0;
+		@include section-divider();
+	}
+}
```

Comment:
All of these styles should go in the parent element, because they deal with the edge of the component.

Reply:
I mean, they should go in the CartComponent.